test/controllers/EvidenceQuestionController.test.ts

```diff
@@ -0,0 +1,84 @@
+import 'reflect-metadata';
+
+import { expect } from 'chai';
+import { MOVED_TEMPORARILY, OK, UNPROCESSABLE_ENTITY } from 'http-status-codes';
+import request from 'supertest';
+
+import 'app/controllers/EvidenceRemovalController';
+import { Appeal } from 'app/models/Appeal';
+import { YesNo } from 'app/models/fields/YesNo';
+import { EVIDENCE_QUESTION_URI} from 'app/utils/Paths';
+
+import { createApp } from 'test/ApplicationFactory';
+
+const createAppeal = (): Appeal => {
+    return {
+        penaltyIdentifier: {
+            companyNumber: '00345567',
+            penaltyReference: 'A00000001',
+        },
+        reasons: {
+            other: {
+                title: 'I have reasons',
+                description: 'They are legit',
+            }
+        }
+    };
+};
+
+
+describe('EvidenceQuestionController', () => {
+
+    describe('GET request', () => {
+
+        it('should return 200 when accessing evidence question page', async () => {
+            const app = createApp({appeal: createAppeal()});
+
+            await request(app).get(`${EVIDENCE_QUESTION_URI}`)
```

Comment:
You do not need to use string interpolation here. Just do `.get(EVIDENCE_QUESTION_URI)`